appveyor.yml

```diff
@@ -15,8 +15,8 @@ environment:
     - nodejs_version: "5"
 
 install:
-  - IF %nodejs_version% EQU 0.10 npm -g install npm@2
-  - IF %nodejs_version% EQU 0.10 set PATH=%APPDATA%\npm;%PATH%
+  - npm -g install npm@latest
```

Comment:
npm install EPERM errors frequently occured on appveyor, and it seems that this error can be fixed by using npm3.

ref. https://github.com/npm/npm/issues/9696